kube-core/src/params.rs

```diff
@@ -389,6 +418,21 @@ mod test {
         //println!("ser is: {}", ser);
         assert_eq!(ser, "{\"dryRun\":[\"All\"]}");
     }
+
+    #[test]
+    fn delete_param_constructors() {
+        let dp_background = DeleteParams::background();
+        let ser = serde_json::to_string(&dp_background).unwrap();
+        assert_eq!(ser, "{\"propagationPolicy\":\"Background\"}");
+
+        let dp_foreground = DeleteParams::foreground();
+        let ser = serde_json::to_string(&dp_foreground).unwrap();
+        assert_eq!(ser, "{\"propagationPolicy\":\"Foreground\"}");
+
+        let dp_orphan = DeleteParams::orphan();
+        let ser = serde_json::to_string(&dp_orphan).unwrap();
+        assert_eq!(ser, "{\"propagationPolicy\":\"Orphan\"}");
+    }
```

Comment:
Nit: Might want to compare `serde_json::to_value` rather than `to_string`, since the formatting isn't really crucial to test here.

Reply:
Good point! Fixed in 79e3066b68f1a2991ed4